contrib/mkimage-arch.sh

```diff
@@ -37,16 +51,30 @@ PKGIGNORE=(
     openresolv
     pciutils
     pcmciautils
+    psmisc
     reiserfsprogs
     s-nail
+    sysfsutils
     systemd-sysvcompat
     usbutils
     vi
+    which
     xfsprogs
 )
+
+PKGREMOVE=(
+    gawk
```

Comment:
Looks like this is using spaces for indentation, but tabs should be used in shell scripts; could you update that?

Reply:
updated